### PR TITLE
added the language strings and documentation for the new run-from-lin…

### LINKED ISSUE #38: Run from line/cursor command
### DESCRIPTION
**What does your feature request improve on? Please describe.**
When debugging a certain section it would be very useful to be able to run the program from a certain starting point. Obviously this would only work for scripts and would add value to the script data type.

**Describe the solution you'd like**
Besides the `run` command I would expect to see a `run from cursor` command.
This could also be an option when right clicking in the code or a graphical option on the left side column where the line numbers are.

**Additional context**

(*Do not write below this line*)

-----------------------------------------------------
### ANALYSIS
This requires a set of changes:

- Providing the interface to start from a selected line in `void Script::openScript(string& _sScriptFileName)` as well as the command line interface like `start SCRIPT -set atline=LINE`
- Provide the corresponding event handler and a short cut, e.g. `SHIFT+F5`
- Ensure that the selected line is not a continued line (then search the start of the continuation)
- Ensure that the selected line is not within a control flow block (then search the start of the control flow block)
- Update the documentation wherever necessary

### IMPLEMENTATION STEPS
(*see also our [Wiki for implementation guidelines](https://github.com/numeredev/NumeRe/wiki/HowTo:-Contribute-Code)*)
- Implement the necessary changes in a new branch created here on GitHub
- Test your implementation

### DOCUMENTATION STEPS
(*see also our [Wiki for further information](https://github.com/numeredev/NumeRe/wiki/HowTo:-Create-Language-Strings)*)
- Update the changes log
- Add comments to your implementation
- Add Doxygen documentation comments
- Create or update the documentation articles (`*.NHLP` and `*.NDB` files, if needed)
- Update the language strings (`*.NLNG` files, if needed)

### PULL REQUEST
- Create a pull request for your changes
- Fill out the template
- Assign @numere-org/maintainers as reviewers

### PR DESCRIPTION
**Pull request by raphael-sys**
### SUMMARY
Implements necessary language changes.
Resolves numere-org/NumeRe#38

*Reviewers:* @numere-org/maintainers

### DOCUMENTATION
* **Documentation articles:**
    * [x] corresponding documentation articles updated
    * [ ] new documentation articles created
    * [ ] not needed
* **Language files:**
    * [x] corresponding language files updated
    * [ ] not needed

### CODE REVIEW

File: de-DE/docs/script.nhlp [hunk 1]
```diff
@@ -24,10 +24,14 @@
 		<keyword>inclu</keyword>
 		<keyword>script</keyword>
 		<keyword>start</keyword>
+		<keyword>from line</keyword>
```
Comment (numeredev) [inline de-DE/docs/script.nhlp]: Whitespaces do not work here. Use the new option to enable searching for the entry

File: en-GB/docs/script.nhlp [hunk 2]
```diff
@@ -24,10 +24,14 @@
 		<keyword>inclu</keyword>
 		<keyword>script</keyword>
 		<keyword>start</keyword>
+		<keyword>from line</keyword>
```
Comment (numeredev) [inline en-GB/docs/script.nhlp]: Same here

**Review by numeredev — CHANGES_REQUESTED** [review]
You might also want to add `fromline` to the option keywords (in syntaxelements.nlng) as well as to add the language string for this option itself (in numere.nlng)

**Comment by raphael-sys** [comment]
> You might also want to add `fromline` to the option keywords (in syntaxelements.nlng) as well as to add the language string for this option itself (in numere.nlng)

What would the return type be for this line?
'GUI_EDITOR_CALLTIP_OPT_FROMLINE=fromline=??? [%%GUI_EDITOR_CALLTIP_COMMON_OPTIONFOR%% start]'

**Review by numeredev — APPROVED** [review]
Changes accepted

**Pull request merged**